### ISSUE TITLE
class and click are mutually exclusive

### ISSUE THREAD

**Issue by ssbarnea** [mermaid-js/mermaid#1213]
I discovered that I cannot use `class` and `click` directives on the same nodes because only one of them will take effect. 

```mermaid
graph LR;
    foo
    classDef core fill:#f9f,stroke:#333,stroke-width:1px;
    class foo core;
    click foo "https://github.com" "foo!"
```

**Comment by GDFaber** [comment]
Hi @ssbarnea, thank you for reaching out. This is a bug which is currently being fixed in #1212.

**Comment by GDFaber** [comment]
I just tested your example with the PR for #1212 to be sure it solves the problem (which it does). Yould you agree on clising this issue as a duplicate?
